backend/environment/env.content-publishing-service.template

```diff
@@ -12,8 +12,8 @@ IPFS_ENDPOINT="http://kubo_ipfs:5001"
 # IPFS_BASIC_AUTH_SECRET=
 
 # IPFS gateway URL. '[CID]' is a token that will be replaced with an actual content ID
-# IPFS_GATEWAY_URL="https://ipfs.io/ipfs/[CID]"
-IPFS_GATEWAY_URL="http://kubo_ipfs:8080/ipfs/[CID]"
+IPFS_GATEWAY_URL="https://ipfs.io/ipfs/[CID]"
```

Comment:
These env files should probably go away as long as the ENVIRONMENT.md has good enough docs?

Reply:
Yes; the only ones needed are the 2 for `social-app-backend`, since the rest have moved inside the docker-compose files.